lib/cli/blueprints/app/files/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "<%= name %>",
   "version": "0.0.1",
   "scripts": {
-    "start": "index.js",
-    "test": "./node_modules/.bin/denali test"
+    "start": "denali server",
```

Comment:
No, local installation of modules with a bin defined will automatically put those modules in the path for npm scripts.